news/5407.documentation

```diff
@@ -0,0 +1 @@
+Solved Edit breaks by broken Short Names @pranayjoshi
```

Comment:
This file should be renamed `5407.bugfix` because it fixes a bug. See https://6.docs.plone.org/contributing/index.html#change-log-entry

Also we like descriptive descriptions in our change logs. "Fixed or Solved issue title" is useless because it does not describe the bug and its fixed state. See item 2 in https://6.docs.plone.org/contributing/first-time.html#create-a-pull-request-from-your-fork.

```suggestion
When you add a page, only lowercase letters (a-z) without accents, numbers (0-9), and the characters "-", "_", and "." are enforced for the short name. Also after creating the page, you can now edit it. Previously you could enter invalid characters and create the page, and then you could not edit it. @pranayjoshi
```